src/Controller/OverviewController.php

```diff
@@ -222,6 +226,67 @@ public function update_category(Request $request, $id): Response
 
   }
 
+  /**
+   * This should be the main page that everyone should see. Every user should be able to see this page and everything
+   * on it. This will be modified more clearly from it's current state. Currently
+   * being used as a testing stage for database outputs.
+   * 
+   * @author Daniel Boling
+   * @return rendered moderation_announcements.html.twig
+   * 
+   * @Route("/moderation/announcements", name="moderation_announcements")
+   * @IsGranted("ROLE_MODERATOR")
+   */
+  public function moderation_announcements(): Response
+  {
+
+    $announcement = $this->getDoctrine()
+      // inits the database and table Announcements;
+      ->getRepository(Announcement::class)
+      ->find_today();
```

Comment:
You'll probably need to change this to `find_today($approval=0)` or it won't find any un-approved announcements.  This is the kind of change that a test should catch.  Do you have tests checking whether they can find un-approved announcements?

Reply:
I don't have any controller tests yet. I only have tests dedicated to checking that the database and query functions work properly.